defaults/main.yml

```diff
@@ -8,9 +11,13 @@ sympa_template_lists: []
 #  ssl: true
 #  queue_transport: sympa
 #  bouncequeue_transport: sympabounce
+
+## Database variables
+sympa_db_type: mysql             # mysql or PostgreSQL
+sympa_install_db_package: False  # yes for this role to install mysql or postgres, no if installed elsewhere 
 sympa_db_name: sympa
 sympa_db_host: localhost
-sympa_db_port: 3306
+sympa_db_port: "{{ 5432 if sympa_db_type == 'PostgreSQL' else 3306 }}"
```

Comment:
This new commit will problably solve [this stucking problem](https://github.com/stuvusIT/sympa/pull/11#discussion_r383489711).